### DISCUSSION TITLE
Bug: Dropdown Menus don't Close on Loosing Focus

### DISCUSSION THREAD

**Discussion by christopher-besch** [community/community#8601] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/57909184/145442219-3ea4fa54-f482-4f42-8674-6a2df215a7a8.png)
These dropdown menus don't close when clicking anywhere else.

Viewing on: Mozilla/5.0 (X11; Linux x86_64; rv:95.0) Gecko/20100101 Firefox/95.0

**Comment by brianlovin** [comment]
Thanks for the heads up! I've passed this along to the team to investigate 👍
